src/scss/skin-modern/_skin-tv.scss

```diff
@@ -0,0 +1,102 @@
+@import 'variables';
+@import 'mixins';
+
+.#{$prefix}-ui-skin-tv {
+  $overscan-margin: 2%;
+  $large-size: 3.5vh;
+  $medium-large-size: 3vh;
+  $medium-size: 2vh;
+
+  font-size: $medium-size;
+
+  .#{$prefix}-ui-titlebar {
+    > .#{$prefix}-container-wrapper {
+      margin: $overscan-margin $overscan-margin $medium-size;
+      width: 100% - 2 * $overscan-margin;
+    }
+  }
+
+  .#{$prefix}-ui-titlebar-top {
+    margin-bottom: 1vh;
+
+    > .#{$prefix}-container-wrapper {
+      width: 100%;
+      font-size: $large-size;
+
+      display: inline-flex;
```

Comment:
There are a bunch of warnings reported by GitHub Actions - I assume linting errors because of wrong sort order. Would be nice to get them fixed.

Reply:
There always have been quite some SASS linting WARNINGS for a long time, GH Actions just makes some of them more prominent now 😉 
I fixed quite a lot of them in the last few commits, a few are still there (they aren't as easy to fix - if we resolve the nesting ones in a straight forward way, we seem to exchange that warning with a nesting depth warning. Might need more care and testing). I'd like to keep that out of this PR if possible.

Reply:
Ok, I thought since we're adding a new file here, it would make sense to already do it "properly" :)
Thanks for fixing them!

We can also think about adjusting linting rules if we feel like they don't make sense for us.

Reply:
Totally get it and usually agree.

I'm not in the position to recommend changing linting rules, but maybe you can discuss internally and let me know the outcome, happy to invest some time later on if I can.